### ISSUE TITLE
not able to update comfy portable

### ISSUE THREAD

**Issue by sumeetprashant** [comfyanonymous/ComfyUI#4711]
### Your question

![image](https://github.com/user-attachments/assets/dc16cc20-4efe-4f7a-8a74-83267b93fbae)

### Logs

```powershell
stashing current changes
Traceback (most recent call last):
  File "Z:\ComfyUI\ComfyUI_windows_portable\update\update.py", line 53, in <module>
    repo.stash(ident)
  File "Z:\ComfyUI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pygit2\repository.py", line 1080, in stash
    check_error(err)
  File "Z:\ComfyUI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pygit2\errors.py", line 66, in check_error
    raise GitError(message)
_pygit2.GitError: path too long: 'Z:/ComfyUI/ComfyUI_windows_portable/ComfyUI/input/TheAramintaExperiment_Cv5_(a masterwork of maximalist fluid art_1.0), (translucent) macro photography by RHADS and Alessio Albi, (extreme details, beautiful delicate texture), interlaced orange and yellow flow.png'
Press any key to continue . . .
```

**Comment by mcmonkey4eva** [comment]
`GitError: path too long`
lol
quickfix: go rename that file to something shorter

proper fix: probably this: https://www.backupery.com/how-to-enable-ntfs-long-paths-in-windows/

**Comment by sumeetprashant** [comment]
haha..
I did do those things..  
mentioned here: proper fix: probably this: https://www.backupery.com/how-to-enable-ntfs-long-paths-in-windows/
before posting here for help..

I don't even have that file in my input folder
I even made a fresh empty input folder 
and still get the same error..

could it be something else.. 
this is my second portable comfy that is giving the same error and refuses to update.
when I try updating python dependencies it updates but comfy fails to update everytime.

**Comment by sumeetprashant** [comment]
Hey
I was able to update comfy finally.
Initially I made a jew 'input' folder and just renamed the old one as 'input_old'.
Even then comfy was cosidering 'input_old' as main input folder and throwing out the same error.

Then I deleted the 'input_old' and also cache from main  _pychache_ folder.
Don't know if deleting cache helped or not.

But after all this comfy was able to get updated.

So we can close this.
Thanks for helping make sure that this was just the name thing. It drove me crazy for a day or two.

Cheers
